lightblock.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"github.com/btcsuite/btcd/blockchain"
+	"github.com/btcsuite/btcd/wire"
+)
+
+type LightBlock struct {
+	height  int32
+	header  *wire.BlockHeader
+	lcStore Store
```

Comment:
why do we need to have store here?

Reply:
I want to separate the storage handle actions and logic verification handle actions. When we move logic to cosmos, we only need to implement the `store` compatible for cosmos without update any external logic for light client.

Reply:
Let's remove the store from here. I think it's not needed here. LightBlock should be used as a type, and verifier should use store.

Reply:
Sorry I think I give the wrong answer! I thought you asked about `store` in `BTCLightClient`. In Lightblock, new need store because btcd need jump back to a few previous blocks (please check [here](https://github.com/btcsuite/btcd/blob/master/blockchain/interfaces.go#L54)).  We use store field to help this work possible.

Reply:
Done!